config/input-resource-verify.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@romani @kukreja-vikramaditya After confirming with cache file from main config run.

`xdoc`, `site, is controlled by main config, so it shouldn't be scanned by "resource" config.
`.ci` folder is ok to add for this PR.

> config

@romani I will create a new issue for this, but if `config` file is scanned for violations and a violation is found, it will invalidate entire cachefile even if config is never changed.